src/odemis/driver/test/simcam_test.py

```diff
@@ -31,18 +31,28 @@
 
 CLASS = simcam.Camera
 KWARGS_FOCUS = {"name": "focus", "role": "overview-focus", "axes": ["z"], "ranges": {"z": [0, 0.012]}}
-KWARGS = dict(name="camera", role="overview", image="simcam-fake-overview.h5")
```

Comment:
Please still test at list a little bit with this RGB image.
I'd suggest to have a separate class `TestSimCamRGB`, which uses these arguments.